src/index.js

```diff
@@ -1 +1,32 @@
 import './style.css';
+import { fetchData, postScore } from './modules/get.js';
+import displayData from './modules/data.js';
+
+const refresh = document.querySelector('#refresh');
+const submit = document.querySelector('#cll');
+
+window.onload = async () => {
+  fetchData();
+  const data = await fetchData();
+  displayData(data);
+};
+submit.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const user = document.getElementById('user');
+  const score = document.getElementById('score');
+  if (user.value && score.value) {
+    await postScore(user.value, score.value);
+    user.value = '';
+    score.value = '';
+    await fetchData();
+    const data = await fetchData();
+    displayData(data);
+  }
+});
+
+refresh.addEventListener('click', async () => {
+  // fetchData();
```

Comment:
- [ ] Kindly remove commented code so as to have a cleaner codebase

Reply:
Hello @Ghiftee I think I left the comments by mistake thanks for the review and response.